tests/dragonfly/replication_test.py

```diff
@@ -984,7 +984,7 @@ async def test_role_command(df_local_factory, n_keys=20):
     # This tests that we react fast to socket shutdowns and don't hang on
     # things like the ACK or execution fibers.
     master.stop()
-    await asyncio.sleep(0.1)
+    await asyncio.sleep(0.3)
```

Comment:
This regressed, so I used a lower interval for `acl-checker` (200ms instead of 1s). we can fine tune this if this ever becomes a problem but I don't see anyother regressions so we should be ok

Reply:
why sending replconf acl check every 1 second had an affect on this?